src/main/resources/attackvectors/UnionBasedSQLInjectionVulnerabilityPayload.properties

```diff
@@ -0,0 +1,8 @@
+UNION_BASED_SQL_INJECTION_PAYLOAD_LEVEL_1=By sending a query with <i>UNION</i> word we can gain data about users or other internal information from the databases. An example for this is:<br/>\
+<i>5 UNION SELECT * FROM users WHERE id=1</i><br/>\
+If you insert this in the textbox, you will get information about a user from the table.
+
+UNION_BASED_SQL_INJECTION_PAYLOAD_LEVEL_2=By sending a query with <i>UNION</i> word we can gain data about users or other internal information from the databases. Previous query does not work because additional security was added - value of <i>id</i> parameter is surrounded by quote in that way <i>id='value'</i> <br/>\
```

Comment:
don't refer previous query as it is shown to a different level. Instead mention the query that doesn't work.